src/v/config/configuration.cc

```diff
@@ -99,6 +99,19 @@ configuration::configuration()
       "Interval for which all coprocessor offsets are flushed to disk",
       required::no,
       300000ms) // five minutes
+  , enable_v8(*this, "enable_v8", "Enable v8 engine", required::no, false)
```

Comment:
Do we need to use enable_inline_wasm? (@senior7515 )
Because at this moment we have only data-policy, but in future we want add inline-wasm like new coproc "executor" (the same as async)